pkg/metrics/metrics.go

```diff
@@ -54,7 +54,16 @@ var (
 			Subsystem: subsystemName,
 			Name:      "pending_workloads",
 			Help:      "Number of pending workloads, per queue and cluster_queue.",
-		}, []string{"cluster_queue", "queue"})
+		}, []string{"cluster_queue", "queue"},
+	)
+
+	AdmittedActiveWorkloads = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: subsystemName,
+			Name:      "admitted_active_workloads",
+			Help:      "Number of admitted workloads that are active (unsuspended and not finished)",
+		}, []string{"cluster_queue", "queue"},
```

Comment:
Since users can create as many queues as they wish, aren't we concerned about this dimension getting too big? do you remember what the limits where?

Reply:
There is no limit. The more queues there are, the more memory we use. Let me try to get more information on how bad this could be for a gauge.

Note that when a queue or clusterQueue is removed, I'm deleting the label values .

Reply:
We chatted offline and concluded that it's best to only keep track of active workloads per clusterqueue, similarly to #293